molecule/shift-stack/molecule.yml

```diff
@@ -0,0 +1,33 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: vagrant
+  provider:
+    name: libvirt
+  provision: no
+  parallel: true
+  default_box: 'generic/centos9s'
+platforms:
+  - name: crc
+    memory: 20480
+    cpus: 8
```

Comment:
would be nice to have these (auto)matching ``crc_*`` ones

Reply:
nope that is intentionaly not the same.
the platform ones must be larger as this is using nested virt.

i had considered using a prepare playbook to avoid that but for now i cant run crc on my host os so I'm going with nested vms.